beacon-chain/operations/attestations/metrics.go

```diff
@@ -31,8 +19,3 @@ var (
 		Help: "The number of expired and deleted block attestations in the pool.",
 	})
 )
-
-func (s *Service) updateMetrics() {
```

Comment:
this is pretty useful, I am inclined to keep this. I would like to know the count through our metrics

Reply:
agreed